integration-tests/test/fee-payment.spec.ts

```diff
@@ -27,8 +27,10 @@ describe('Fee Payment Integration Tests', async () => {
       other,
       utils.parseEther('0.5')
     )
-    const executionGas = await (env.ovmEth
-      .provider as any).send('eth_estimateExecutionGas', [tx, true])
+    const executionGas = await (env.ovmEth.provider as any).send(
+      'eth_estimateExecutionGas',
+      [tx, true]
+    )
```

Comment:
why do these linting changes occur?

Reply:
Yes, (right now) parts of the tooling break other parts of the tooling, so if you run everything, one thing breaks the other and back and forth. There are two formatting systems, and they have different ideas of what the code should look like